mavimage/image.py

```diff
@@ -0,0 +1,132 @@
+"""Handle Image class and helper functions
+Lauren McIntire"""
+
+from fractions import Fraction
+from datetime import datetime
+import io
+import PIL.Image
+import piexif
+from .gps import GPSRecord
+
+
+class Image:
+    """
```

Comment:
Use [Sphinx](https://pythonhosted.org/an_example_pypi_project/sphinx.html), [NumPy](http://www.sphinx-doc.org/en/stable/ext/example_numpy.html), or [Google](http://www.sphinx-doc.org/en/stable/ext/example_google.html#example-google) style docstrings.  The choice is yours but choose wisely as you need to be consistent.